### ISSUE TITLE
Option to display an input type text with the feed URL

### ISSUE THREAD

**Issue by AntoineTurmel** [nt1m/livemarks#56]
I used to copy-paste feeds URL from url bar of Firefox but with this webext it has:
```
moz-extension://a4f28a05-bf56-441f-83fc-76278943027f/pages/subscribe/subscribe.html?https%3A//github.com/AntoineTurmel.private.atom%3Ftoken%3DAAHR-CAZrIeacTJJSJS1BvROiXg9kYbsks655CLUwA%3D%3D
```
It could be nice to have an input type text with the full RSS feed URL on the feed preview :)

**Comment by evilpie** [comment]
Implemented in #53, but not released yet.

**Comment by AntoineTurmel** [comment]
Nice !!!
